test/smoke/testApps/TraceLogBackUsingAgent/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/TraceLogBackTest.java

```diff
@@ -58,7 +58,6 @@ public void testTraceLogBack() throws Exception {
     assertEquals("This is logback warn.", md1.getMessage());
     assertEquals(SeverityLevel.Warning, md1.getSeverityLevel());
     assertEquals("Logger", md1.getProperties().get("SourceType"));
-    assertEquals("WARN", md1.getProperties().get("LoggingLevel"));
```

Comment:
can you find out what .NET does, if they add some kind of `level` property to customDimensions, or if their users rely only on "severity level" kusto column?

Reply:
Confirmed with Cijo, they don't send logginglevel.. only severityLevel.